project3/styles.css

```diff
@@ -0,0 +1,285 @@
+* {
+    box-sizing: border-box;
+    transition: 400ms;
+}
+
+
+body {
+    background-color: #163346;
+    margin: auto;
+
+}
+
+div {
```

Comment:
In general, you should avoid putting such specific styles on a base `div` -- it's these styles, for example, that are adding the background color behind your SHIELD logo SVG. Styles on base elements like a paragraph of text input can be okay -- it's likely you want most of your paragraphs and text inputs to look alike, but `div` is _too_ generic and therefore causes problems like this, even in small projects.